### DISCUSSION TITLE
Object detection application malloc() error while running

### DISCUSSION THREAD

**Discussion by AmarRobo** [AxisCommunications/discussions#179] [General]
Object Detection app needs several start and stops before it gets running successfully. It usually takes 8 to 10 times of restarting the app before the malloc() error resolves. I have additional SD card on the camera. Is there any particular reason for this behavior? 
I am hoping to have an object detection app that runs on start up and this behavior will most probably make it impossible

**Comment by mattias-kindborg-at-work** [comment]
Are we talking about one of the examples in the [CV SDK repository](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples) or in the [Native SDK repository](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples)? If *"yes"* I would advice you to ask the question in that specific repository.

**Reply by AmarRobo** [reply]
Ok. Thanks
